Dockerfile

```
@@ -0,0 +1,47 @@
+RUN groupadd --system app \
+    && useradd --system --gid app --create-home app \
+    && chown -R app:app /app
+
+USER app
+
+WORKDIR /app/backend
+
+EXPOSE 8000
+# … earlier Dockerfile lines …
+
+RUN apt-get update \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)